lib/random-schedule.js

```diff
@@ -0,0 +1,74 @@
+const Random = require('random-js')
+const random = new Random(Random.engines.browserCrypto)
+const Scheduler = require('azure-arm-scheduler')
+const randString = require('crypto-random-string')
+
+exports.calcSeconds = function (timestamp) {
```

Comment:
is there a library that will do this date handling/maths for us? in my experience this is usually tough to get right and i'd feel better about using a trusted source rather than rolling our own